README.md

```diff
@@ -1,2 +1,136 @@
-# Hello_Microverse
-This is the first Microverse Project
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>README</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+-- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> In this project, you will set up a "Hello Microverse" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. 
+
+**Hello-Microverse** is a the first project built by the developer who is in the early stages of learning how to code. It desmostrate how to master all of the tools and best practices and how to code remotely to your github repository.
+
+Furthermore, you will learn how to use linters to fix errors in your code and make sure that your code is clean.
+
+## 🛠 Built With  <a name="built-with"></a>
+
```

Comment:
Great job adding the README template 👍 , I think it will be good if you add the languages used in building this project. e.g HTML, CSS etc 😊 ☺️